src/community/smart-data-loader/src/main/java/org/geoserver/smartdataloader/domain/IndexedDomainModelVisitor.java

```diff
@@ -0,0 +1,10 @@
+package org.geoserver.smartdataloader.domain;
+
+public interface IndexedDomainModelVisitor extends DomainModelVisitor {
```

Comment:
A bit of javadoc explaining what this interface does would not hurt.... (and the methods)

Reply:
Right, adding.